src/stories/migration.mdx

```diff
@@ -14,6 +14,21 @@ Breaking Changes
     - `unpinText` and `pinText` props moved to `textStrings.pin` & `textStrings.unpin`
   - Data table
     - 'ellipsis' prop is removed
+- Added component props
+  - Toggle Button
+    - `label` prop is added
+- Removed component slots
+  - Checkbox Group
+    - `label` slot moved to `label` prop
+  - Dropdown
+    - `label` slot moved to `label` prop
+- Added component slots
+  - Checkbox Group
+    - `tooltip` slot is added
+  - Dropdown
+    - `tooltip` slot is added
+  - Toggle Button
+    - `unnamed` slot is for tooltip
```

Comment:
Pull in the latest from main containing Vijeta's changes, and add your notes under the section she created.